MnAu2/mnau2_fm.scf.in

```diff
@@ -0,0 +1,32 @@
+ &control
+    calculation = 'scf',
+    pseudo_dir = '../pseudo',
+    outdir='../temp'
+    prefix='mnau2',
+ /
+ &system
+    ibrav=  0, 
+    nat=  3, 
+    ntyp= 3,
+    ecutwfc = 60.0, ecutrho = 800.0,
+    occupations='smearing', smearing='mv', degauss=0.02,
+    nspin=2, 
+    starting_magnetization(1)= 0.8,
+ /
+ &electrons
+    mixing_beta = 0.3 
+ /
+CELL_PARAMETERS
+ 3.3701291084         0.0000000000         0.0000000000
+ 0.0000000000         3.3701291084         0.0000000000
+-1.6850646988        -1.6850646988         4.3794687067
+ATOMIC_SPECIES
+ Mn    1.  Mn.pbe-sp-hgh.UPF
```

Comment:
This pseudopotential may be the reason that you're unable to stabilize magnetism. It's a norm-conserving pseudopotential, and requires large energy cutoffs (parameter `ecutwfc`), probably larger than the `ecutwfc` used here. You may be able to stabilize magnetism if you increase it.

Alternatively, you can use Projector Augmented Wave (PAW) potentials instead. They yield similar results to the pseudopotential files, but don't require very large cutoffs. I found PAW potentials for Mn and Au, edited the input file (in addition to setting the new pseudopotentials, I set the `occuptations` keyword equal to `tetrahedra`, it's generally preferable to use this when doing self-consistent calculations if it's available. It required that I make the kmesh 8×8×8 so that it's equal on all sides), and re-ran the calculation, and I stabilized magnetism. Here's my files:

[QE_MnAu2_new_pseudos.tar.gz](https://github.com/Mason-DataMaterials/spyns/files/1690949/QE_MnAu2_new_pseudos.tar.gz)